.golangci.example.yml

```diff
@@ -186,6 +186,10 @@ linters-settings:
         paramsOnly: true
       rangeValCopy:
         sizeThreshold: 32
+  godox:
+    keywords:
+      - FIXME
+      - TODO
```

Comment:
please use some practical example - it would be practical if it would be different from the default keywords

Reply:
Added examples and comments.